app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Users::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  # def new
+  #   super
+  # end
+
+  # POST /resource/sign_in
+
+  # DELETE /resource/sign_out
```

Comment:
- [ ] It will be nice to remove commented code from your code base as this makes your code base dirty and less readable.